Sources/SymDiff/source/Options.cs

```diff
@@ -101,6 +101,17 @@ public enum DAC_ENCODING_OPT { DAC_LINEAR, DAC_NORMAL };
 
         #endregion 
 
+        #region Supporting custom heap comparison predicates
+
+        // Is a custom heap comparison method specified?
+        public static bool CustomHeapComparison = false;
```

Comment:
Alternatively, we could set it up so that `GenerateComparison` (with a slightly different type) was, by default, a reference to `EqualityReduction`, and was always called. To customize it, we'd just replace that default binding with a custom one. It might take too much refactoring to do that at the moment, but I at least want to record this possibility somewhere.